src/postgres/7-seed.sql

```diff
@@ -344,6 +345,13 @@ INSERT INTO privilege VALUES ('QUESTION_DELETE');
 -- RolePrivilege(edprole_id, privilege) Administrator
 INSERT INTO edprole_privilege VALUES ('75605ac9-bf65-4dec-8458-93e018dcca97', 'ADMIN');
 
+-- RolePrivilege(edprole_id, privilege) UWG Member
+INSERT INTO edprole_privilege VALUES ('19ac227b-e96c-46fa-a378-cf82c461b669', 'REQUEST_READ');
+INSERT INTO edprole_privilege VALUES ('19ac227b-e96c-46fa-a378-cf82c461b669', 'REQUEST_REVIEW');
+INSERT INTO edprole_privilege VALUES ('a5b4947a-67d2-434e-9889-59c2fad39676', 'NOTE_NEW');
```

Comment:
Please fix the edprole_id.  They should be `19ac227b-e96c-46fa-a378-cf82c461b669`